### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by bossmonkey98**
Homepage Update

Apex Store — Mozilla Firefox 15-02-2022 14_44_48
Apex Store — Mozilla Firefox 15-02-2022 15_05_14
Apex Store — Mozilla Firefox 15-02-2022 15_05_23

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,8 +41,67 @@
...
+        </div>
+        <br>
+        <h1>LEGENDARY COLLECTION</h1>
+        <br>
+        <div class="home-products">
+            <div class="vertical-card">
+                <img class="vertical-card-img" src="./components/assets/images/prdouct1.webp" alt="">
+                <span class="card-badge">New</span>
+                <div class="vertical-card-content">
+                    <br>
```
Comment (purvasharmadev) [inline index.html:56]: Avoid using br tags

**Review by purvasharmadev — APPROVED** [review]
Everything looks great. Nice UI. Just add some comments and spacing for cleaner code.
Nice work!!

**Review by DivyanshSareen — APPROVED** [review]
Looks Good. good to merge

File: index.html (hunk 1 above)
Comment (DivyanshSareen) [reply index.html:56]: +1

Comment (bossmonkey98) [reply index.html:56]: Okay

**Pull request merged**